sdk/digitaltwins/azure-digitaltwins-core/src/main/java/com/azure/digitaltwins/core/models/ComponentMetadata.java

```diff
@@ -8,12 +8,18 @@
 import com.fasterxml.jackson.annotation.JsonAnySetter;
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
 import java.util.HashMap;
 import java.util.Map;
 
 /**
  * An optional helper class for deserializing a digital twin
+ * <p>
```

Comment:
where does this paragraph ends?

Reply:
This is just something that Javadocs allow. It just means there is a paragraph split at this point

Reply:
I mean ... why not just have a fully valid html tag?

Reply:
HTML is loosey goosey when it comes to some tags, like `<p>`